buddyboss/buddyboss-platform
# Activity > Reactions

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #270 

### How to test the changes in this Pull Request:

1. Go to Buddy Boss -> Settings -> Activity
2. Mark checkbox checked "Allow for enhanced "Reaction" emojis"
3. See the reactions buttons at every activity 

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
